specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json

```diff
@@ -1647,6 +1650,18 @@
         "tier": {
           "type": "string",
           "description": "Performance tier of the disk (e.g, P4, S10) as described here: https://azure.microsoft.com/en-us/pricing/details/managed-disks/. Does not apply to Ultra disks."
+        },
+        "securityType": {
+          "type": "string",
+          "description": "Specifies the SecurityType of the VM. It is set as SecureBoot for TrustedVMs.",
+          "enum": [
+            "None",
+            "SecureBoot"
+          ],
+          "x-ms-enum": {
+            "name": "SecurityTypes",
+            "modelAsString": false
```

Comment:
All enums in our swagger should have this flag true. Otherwise adding a new enum member in the future will be a breaking change

Reply:
changed modelAsString to true.